src/main/java/com/commercetools/project/sync/service/CustomObjectService.java

```diff
@@ -0,0 +1,23 @@
+package com.commercetools.project.sync.service;
+
+import com.commercetools.project.sync.model.LastSyncCustomObject;
+import io.sphere.sdk.customobjects.CustomObject;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import javax.annotation.Nonnull;
+
+public interface CustomObjectService {
+  @Nonnull
+  CompletionStage<ZonedDateTime> getCurrentCtpTimestamp();
+
+  @Nonnull
+  CompletionStage<Optional<CustomObject<LastSyncCustomObject>>> getLastSyncCustomObject(
+      @Nonnull final String sourceProjectKey, @Nonnull final String syncModuleName);
+
+  @Nonnull
+  CompletionStage<CustomObject<LastSyncCustomObject>> persistLastSyncCustomObject(
```

Comment:
maybe `updateLastSyncCustomObject`could be a better name, I think we are not persisting it, API does.

Reply:
true you are right. I changed this wording on the PR of the unit tests: https://github.com/commercetools/commercetools-project-sync/pull/25

to `createLastSyncCustomObject` because it's being created/update. Maybe an even better name could be `upsert` to use the same vocab as the platform. WDYT?

Reply:
both of them is ok for me. looks good in #25